src/libs/OptionsListUtils.js

```diff
@@ -56,19 +56,22 @@ function getPersonalDetailsForLogins(logins, personalDetails) {
  * @return {String}
  */
 function getSearchText(report, personalDetailList) {
-    const searchTerms = [];
+    // Use a set because it skips adding duplicates,
+    // which could potentially significantly speed up our regex search if someone has thousands of chats
+    const searchTerms = new Set();
 
     _.each(personalDetailList, (personalDetail) => {
-        searchTerms.push(personalDetail.displayName);
-        searchTerms.push(personalDetail.login);
+        searchTerms.add(personalDetail.displayName);
+        searchTerms.add(personalDetail.login);
     });
 
     if (report) {
-        searchTerms.push(...report.reportName.split(',').map(name => name.trim()));
-        searchTerms.push(...report.participants);
+        searchTerms.add(report.reportName);
+        searchTerms.add(...report.reportName.split(',').map(name => name.trim()));
+        searchTerms.add(...report.participants);
```

Comment:
I think we can't spread array of string into a `Set` 🤔  ?

<img width="483" alt="Screen Shot 2021-05-26 at 4 02 56 PM" src="https://user-images.githubusercontent.com/32969087/119754234-e2105080-be3b-11eb-996f-7e5d276d05e3.png">

Reply:
Ah, didn't know this, thanks for catching this!